src/testing/testpachd/main.go

```diff
@@ -165,12 +150,9 @@ func main() {
 		os.Stdout.Close()
 	}()
 
-	// With pachd started and the config ready, run until the context is done.  Background
-	// errors cause this, as does SIGINT.
-	<-ctx.Done()
-	if err := <-errCh; err != nil {
+	// wait for pachd to complete
```

Comment:
There’s no need to wait for context cancellation, since the run goroutine will close `errCh` when it completes.